cmd/harmony/main.go

```diff
@@ -76,7 +75,7 @@ var (
 	ip               = flag.String("ip", "127.0.0.1", "ip of the node")
 	port             = flag.String("port", "9000", "port of the node.")
 	logFolder        = flag.String("log_folder", "latest", "the folder collecting the logs of this execution")
-	logMaxSize       = flag.Int("log_max_size", 100, "the max size in megabytes of the log file before it gets rotated")
+	logMaxSize       = flag.Int("log_max_size", 100000, "the max size in megabytes of the log file before it gets rotated")
```

Comment:
Please revert this for the time being, for the reasons I posted in the PR above.

Reply:
Or, alternately, make it an option.

Reply:
It's an arg, so you can specify it.

Reply:
Oh sorry, I didn't see that.

Reply:
I'm still not sure people will want that, 1g is a pretty big log file, and if logrotate can't rotate it for you....

Reply:
I guess 10gb might be better yeah

Reply:
I honestly think you may want to just leave it alone to avoid surprises, especially if validator log is going away, there is no more inconsistency, and I can get rid of my own logrotate games.